src/components/Cluster/ClusterDetail/NodePool.js

```diff
@@ -14,6 +13,21 @@ import AvailabilityZonesWrapper from './AvailabilityZonesWrapper';
 import NodePoolDropdownMenu from './NodePoolDropdownMenu';
 import ScaleNodePoolModal from './ScaleNodePoolModal';
 
+const NPViewAndEditName = styled(ViewAndEditName)`
+  input[type='text'] {
+    font-size: 15px;
+    line-height: 1.8em;
+    margin-bottom: 0;
+  }
+  .btn-group {
+    top: 0;
+  }
+  button {
+    font-size: 13px;
+    padding: 4px 10px;
+  }
+`;
+
```

Comment:
Extracted this from the `ViewAndEditName` component, since that should be more generic